src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/PortPropertiesSpec.groovy

```diff
@@ -139,30 +135,32 @@ class PortPropertiesSpec extends HealthCheckSpecification {
         def dstPortDiscoveryOff = disableDiscoveryOnPort(islToManipulate.dstSwitch.dpId, islToManipulate.dstPort)
 
         and: "Bring port up on the src switch"
-        def portUp = antiflap.portUp(islToManipulate.srcSwitch.dpId, islToManipulate.srcPort)
+        antiflap.portUp(islToManipulate.srcSwitch.dpId, islToManipulate.srcPort)
 
         then: "Link is not detected"
         Wrappers.timedLoop(discoveryInterval + WAIT_OFFSET / 2) {
             assert !islUtils.getIslInfo(islToManipulate).isPresent()
         }
 
         when: "Deactivate/activate src switch"
-        def blockData = lockKeeper.knockoutSwitch(sw, RW)
-        def switchStatus
-        Wrappers.wait(discoveryTimeout + WAIT_OFFSET) {
-            switchStatus = northbound.getSwitch(sw.dpId).state
-            assert switchStatus == SwitchChangeType.DEACTIVATED
-        }
-
-        lockKeeper.reviveSwitch(sw, blockData)
-        Wrappers.wait(discoveryInterval + WAIT_OFFSET) {
-            switchStatus = northbound.getSwitch(sw.dpId).state
-            assert switchStatus == SwitchChangeType.ACTIVATED
-            def links = northbound.getAllLinks()
-            (relatedIsls - islToManipulate).forEach {
-                assert islUtils.getIslInfo(links, it).get().state == IslChangeType.DISCOVERED
-            }
-        }
+//        def blockData = lockKeeper.knockoutSwitch(sw, RW)
```

Comment:
do we need this commented code(line 146-163)?

Reply:
Nope. Removed.